pom.xml

```diff
@@ -30,7 +30,7 @@
     <dependency>
       <groupId>org.folio</groupId>
       <artifactId>mod-custom-fields</artifactId>
-      <version>1.0.3</version>
+      <version>1.0.4-SNAPSHOT</version>
```

Comment:
dependency to a snapshot is not permitted
please release the library version and use it here